README.md

```diff
@@ -73,7 +73,7 @@ k3sup was developed to automate what can be a very manual and confusing process
 
 ```sh
 curl -sLS https://get.k3sup.dev | sh
-sudo install k3sup /usr/local/bin/
+sudo mv k3sup /usr/local/bin/
```

Comment:
Install is a valid command, at least on MacOS. Can you Google it?

Reply:
yes you are right 🤦


```
  Copy files and set attributes.
  Copy files (often executable) to a system location like `/usr/local/bin`, give them the appropriate permissions/ownership.
```